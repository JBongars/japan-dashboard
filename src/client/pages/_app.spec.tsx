import * as React from "react";
import * as App from ".";
import { shallow, ShallowWrapper } from "enzyme";

describe("# _App", (): void => {
  it("should render the app", (): void => {
    const wrapper: ShallowWrapper = shallow(<App.default />);

    expect(wrapper).toMatchSnapshot();
  });
});
