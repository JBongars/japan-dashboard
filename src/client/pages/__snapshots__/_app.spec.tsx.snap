// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# _App should render the app 1`] = `ShallowWrapper {}`;
